definition/placetype_counts_by_country.md

```diff
@@ -0,0 +1,257 @@
+iso | country | macroregion | region | macrocounty | county | localadmin | locality | borough | macrohood | neighbourhood | microhood
+-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
```

Comment:
If you add `:` to this 2nd column heading row it'll right align the table cells below:

-- | -- | --: | --: | --: | --: | --: | --: | --: | --: | --: | --:

Reply:
Added in 1b1aa48. Let me know if this formatting works.